frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -34,15 +34,15 @@ export class ResourceSidebarWrapper extends React.Component {
 
     if (!showSidebar) {
       return <div className="co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs">
-        <button className="btn btn-link" onClick={this.toggleSidebar}>
+        <button className="btn btn-link" data-test-id="view-schema" onClick={this.toggleSidebar}>
           <InfoCircleIcon className="co-icon-space-r co-p-has-sidebar__sidebar-link-icon" />{linkLabel}
         </button>
       </div>;
     }
 
     return <div className="co-p-has-sidebar__sidebar co-p-has-sidebar__sidebar--bordered hidden-sm hidden-xs" style={{height}}>
       <div className="co-m-pane__body">
-        <button type="button" className="close" aria-label="Close" onClick={this.toggleSidebar}>
+        <button type="button" className="close" aria-label="Close" data-test-id="close" onClick={this.toggleSidebar}>
```

Comment:
I suggest a more precise value like

```suggestion
        <button type="button" className="close" aria-label="Close" data-test-id="sidebar-close" onClick={this.toggleSidebar}>
```